tajo-plan/src/main/java/org/apache/tajo/plan/rewrite/rules/FilterPushDownRule.java

```diff
@@ -358,7 +353,21 @@ public LogicalNode visitJoin(FilterPushDownContext context, LogicalPlan plan, Lo
       context.pushingDownFilters.removeAll(matched);
     }
 
-    context.pushingDownFilters.addAll(outerJoinFilterEvalsExcludePredication);
+    matched.clear();
```

Comment:
Could add some brief comment about the purpose of the following block? It would be helpful for other code readers.